### PR TITLE
Removing unused code #1

### PR DESCRIPTION
**Pull request by kushti**
In this PR, different unused imports, assignments, arguments are removed

### CODE REVIEW

**Comment by aslesarenko** [comment]
@kushti, please fix the tests.

**Comment by kushti** [comment]
> @kushti, please fix the tests.

fixed

**Review by aslesarenko — CHANGES_REQUESTED** [review]
Looks good, but needs to be merged with v5.0.14-RC, as this is the latest version.

File: interpreter/shared/src/main/scala/sigmastate/serialization/ValueSerializer.scala [hunk 1]
```diff
@@ -256,7 +252,6 @@ object ValueSerializer extends SigmaSerializerCompanion[Value[SType]] {
 
   def printSerInfo(): String = {
     serializerInfo.map { case (_, s) =>
-      val ser = getSerializer(s.opCode)
       s.toString
```
[the remark this answers is not in the document]
Comment (kushti) [reply interpreter/shared/src/main/scala/sigmastate/serialization/ValueSerializer.scala]: assignment is not needed anyway, fixed with a note

**Comment by kushti** [comment]
> Looks good, but needs to be merged with v5.0.14-RC, as this is the latest version.

Merging done. please make another round

**Review by aslesarenko — APPROVED** [review]

**Pull request merged**